specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -549,18 +541,9 @@
           "description": "The time interval, in minutes, at which the LCM checks a pull service to get updated configurations. This value is ignored if the LCM is not configured in pull mode. The default value is 30."
         },
         "rebootIfNeeded": {
-          "type": "string",
+          "type": "boolean",
           "readOnly": true,
-          "default": "False",
-          "description": "Set this to true to automatically reboot the node after a configuration that requires reboot is applied. Otherwise, you will have to manually reboot the node for any configuration that requires it. The default value is false. To use this setting when a reboot condition is enacted by something other than DSC (such as Windows Installer), combine this setting with the xPendingReboot module.",
-          "x-ms-enum": {
-            "name": "rebootIfNeeded",
-            "modelAsString": true
-          },
-          "enum": [
-            "True",
-            "False"
-          ]
+          "description": "Set this to true to automatically reboot the node after a configuration that requires reboot is applied. Otherwise, you will have to manually reboot the node for any configuration that requires it. The default value is false. To use this setting when a reboot condition is enacted by something other than DSC (such as Windows Installer), combine this setting with the xPendingReboot module."
         },
         "configurationModeFrequencyMins": {
           "type": "number",
```

Comment:
Same here. We need to remove the `default` because this property is read-only

Reply:
Fixed. Removed the readonly field. This was by mistake.